install/cupy_builder/_compiler.py

```diff
@@ -72,8 +72,14 @@ def _nvcc_gencode_options(cuda_version: int) -> List[str]:
                          ('compute_80', 'sm_80'),
                          ('compute_86', 'sm_86'),
                          ('compute_89', 'sm_89'),
-                         ('compute_90', 'sm_90'),
-                         'compute_90']
+                         ('compute_90', 'sm_90'),]
+            if cuda_version >= 12080:
+                arch_list += [('compute_100', 'sm_100'),
+                              ('compute_120', 'sm_120'),
+                              'compute_120']
```

Comment:
Might be better to use `compute_100` as a fallback PTX build because it seems there are several CCs in between?
https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#virtual-architecture-feature-list

Reply:
Good idea!
```suggestion
                              'compute_100']
```